src/main/java/uk/gov/companieshouse/web/lfp/controller/lfp/EnterLFPDetailsController.java

```diff
@@ -77,29 +82,38 @@ public String postLFPEnterDetails(@ModelAttribute("enterLFPDetails") @Valid Ente
 
             // If there are no payable late filing penalties either the company does not exist or has no penalties.
             if (payableLateFilingPenalties.isEmpty()) {
+                LOGGER.info("No late filing penalties for company no. "  +  companyNumber
+                        + " and penalty: " +   penaltyNumber + " . Or the company no. does not exist");
                 return UrlBasedViewResolver.REDIRECT_URL_PREFIX + urlGenerator(companyNumber, penaltyNumber) + LFP_NO_PENALTY_FOUND;
             }
 
             // If there is more than one payable penalty.
             if (payableLateFilingPenalties.size() > 1) {
+                LOGGER.info("Online payment unavailable as there is more than one payable penalty. There are " + payableLateFilingPenalties.size() +
+                        " payable penalties for company no. " + companyNumber);
```

Comment:
Sometimes the plus sign for String concatenation, is at the end of the line and sometimes it is in the beginning of the line